src/OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -98,21 +98,26 @@ std::string LiveFunctionsDataView::GetValue(int row, int column) {
   }
 }
 
-std::optional<int> LiveFunctionsDataView::GetSelectedIndex() {
-  return GetRowFromFunctionId(selected_function_id_);
+std::vector<int> LiveFunctionsDataView::GetVisibleSelectedIndices() {
+  std::optional<int> visible_selected_index = GetRowFromFunctionId(selected_function_id_);
+  std::vector<int> visible_selected_indices;
+  if (visible_selected_index.has_value()) {
+    visible_selected_indices.push_back(visible_selected_index.value());
```

Comment:
nit. we could use initializer lists here : "return {visible_selected_index.value()};" and "return {}", I find it clearer and takes up fewer lines.

Reply:
Make sense! Changed accordingly. Thanks!